web/src/Components/Optimization/OptimizationContainer.tsx

```diff
@@ -1,11 +1,11 @@
 import React, { ReactElement, useState } from 'react'
 import OptimizationRunner from './OptimizationRunner'
 import OptimizationResult from './OptimizationResult'
-import { Combination, Products } from '../../Types'
+import { Combination, OptimizationData, Products, ProductsInCombination } from '../../Types'
 
 interface OptimizationContainerProps {
   products: Products
-  addCombination: Function
+  addCombinationsFromOptimization: Function
```

Comment:
addCombination() calls updateCombinationAndFetchBridge()
Inside updateCombinationAndFetchBridge(), setBridges() is used to update the state. 

Because the optimizer returns two combinations, the addCombination would be called two times, which means setBridges() runs two times - I had problems that setBridges() did not finish updating the state before running the second time, which means that only one of the combinations from the optimizer was stored as a bridge (not both)